### PR TITLE
Add *BSD compatibility (use GNU sed).

### PR DESCRIPTION
**Pull request by netchild**

### CODE REVIEW

**Comment by netchild** [comment]
This time without the stupid mistake in the linux case which triggered the message to install sed.

**Comment by itoffshore** [comment]
@netchild many thanks for re-submitting.

I think the `$SED` variable should be enclosed in curly braces as has been [done in `ceph` for FreeBSD `sed` compatibility](https://github.com/ceph/ceph/pull/13903/commits/0a91b76f2f68531a93f5bb790a48f3ba857dbf21)

**Comment by netchild** [comment]
Curly braces added.

File: update-ngxblocker [hunk 1]
```diff
@@ -209,6 +209,17 @@ check_args() {
 }
 
 check_depends() {
+	local OS=$(uname -s)
```
[the remark this answers is not in the document]
Comment (netchild) [reply update-ngxblocker]: changed.

**Pull request merged**